### ISSUE TITLE
Can we eliminate the mediator file?

### ISSUE THREAD

**Issue by rmscheller** [LANDIS-II-Foundation/Library-Initial-Community#7]
We no longer need the file that points to a file for initial communities.  We could simply Parse directly:

protected override IDataset Parse(string csv_file)
        {
            // ReadLandisDataVar();

            Dataset dataset; // = new Dataset();

            //InputVar<string> csv = new InputVar<string>("CSV_File");
            //ReadVar(csv);
            dataset = ReadCSVInputFile(csv_file);

            return dataset;

        }

**Comment by aruzicka555** [comment]
Yes, I have removed it. Now it will parse whatever is passed to the
"InitialCommunities" keyword in the succession scenario files as the
initial community csv. So it will look like the below:

InitialCommunities       ./biomass-succession_InitialCommunities.csv

I recompiled a few repos with this change and the change you made. I
submitted the pull requests for you. The pull requests were to:

Library initial community universal
Extension Biomass succession
support library dll v8
necn succession
extension dgs succession
